### PR TITLE
Hide monaco cursor in Chromatic and Percy

### PR DESCRIPTION
**Pull request by felixfbecker**
Fixes flakiness in integration/Percy tests and prevents it in the future.

### CODE REVIEW

File: .storybook/preview-head.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <style type="text/css">
     @media only percy {
-        .percy-hide {
-            display: none;
-            visibility: hidden;
+        .percy-hide,
+        .monaco-editor .cursor {
+            visibility: hidden !important;
         }
```
Comment (eseliger) [inline .storybook/preview-head.html:6]: is percy actually running over storybooks? I think this whole thing can be removed?

**Review by eseliger — APPROVED** [review]
Change looks good, and approved the chromatic changes. For percy, however, I'm confused about the change and seems to be still a flake? :(

**Comment by felixfbecker** [comment]
I don't see anything flaky in Percy, just the change from `display: none` to `visibility: hidden` (which makes the element invisible, but lets it still affect layout). Do you think we should stick with `display: none` instead?

File: .storybook/preview-head.html (hunk 1 above)
Comment (felixfbecker) [reply .storybook/preview-head.html:6]: You're right, removed.

**Pull request merged**